File: 004.cpp
#include <iostream>
#include <vector>
int main(){
    int H,W; std::cin >> H >> W;
    std::vector<std::vector<int>> A(H, std::vector<int>(W,0));
    std::vector<int> col_sum(W,0), row_sum(H,0);
    for(int h=0; h<H; h++){
        for(int w=0; w<W;w++){
            std::cin >> A[h][w];
            col_sum[w] += A[h][w];
            row_sum[h] += A[h][w];
        }
    }

    std::vector<std::vector<int>> ans(H, std::vector<int>(W,0));
    for(int h=0; h<H; h++){
        for(int w=0; w<W;w++){
            ans[h][w] = col_sum[w] + row_sum[h] - A[h][w];
        }
    }

    for(auto line:ans){ 
        for(auto a:line) std::cout << a << " "; 
        std::cout << std::endl; 
    }
}

File: 004.py
H,W = map(int, input().split())
A = [[int(a) for a in input().split()] for _ in range(H)]
col_sum = [0] * W
row_sum = [0] * H
for h in range(H):
    for w in range(W):
        col_sum[w] += A[h][w]
        row_sum[h] += A[h][w]

ans = [[0]*W for _ in range(H)]
for h in range(H):
    for w in range(W):
        ans[h][w] = col_sum[w] + row_sum[h] - A[h][w]

for a in ans:
    print(*a)



##TLEコード